src/main/java/bio/terra/cli/serialization/userfacing/UFSshKey.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.cli.serialization.userfacing;
+
+import bio.terra.cli.utils.UserIO;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.io.PrintStream;
+
+@JsonDeserialize(builder = UFSshKey.Builder.class)
+public class UFSshKey {
+  public final String privateSshKey;
+  public final String publicSshKey;
+  public final String userEmail;
+
+  private UFSshKey(Builder builder) {
+    this.privateSshKey = builder.privateSshKey;
+    this.publicSshKey = builder.publicSshKey;
+    this.userEmail = builder.userEmail;
+  }
+
+  public static UFSshKey createUFSshKey(String privateKey, String publicKey, String userEmail) {
+    return new Builder()
+        .privateSshKey(privateKey)
+        .publicSshKey(publicKey)
+        .userEmail(userEmail)
+        .build();
+  }
+
+  /** Print out this object in text format. */
+  public void print() {
+    PrintStream OUT = UserIO.getOut();
+    OUT.println("User email: " + userEmail);
+    OUT.println("Private key: ");
+    OUT.println(privateSshKey);
```

Comment:
Yeah I debated on that since the JSON format will need to show both still so that i can access it in a script. I will remove the private ssh key from the output print and add an option to save the key in the ./ssh path.